en/config_rover/ackermann.md

```diff
@@ -133,9 +131,9 @@ To set up [Acro mode](../flight_modes_rover/ackermann.md#acro-mode) configure th
    To tune this parameter:
 
    1. Put the rover in [Acro mode](../flight_modes_rover/ackermann.md#acro-mode) and hold the throttle stick and the right stick at a few different levels for a couple of seconds each.
-   1. Disarm the rover and from the flight log plot the _lateral_acceleration_setpoint_ from [RoverAckermannSetpoint](../msg_docs/RoverAckermannSetpoint.md) and the _measured_lateral_acceleration_ from [RoverAckermannStatus](../msg_docs/RoverAckermannStatus.md) over each other.
-   1. Increase [RA_LAT_ACCEL_P](#RA_LAT_ACCEL_P) if the measured value does not track the setpoint fast enough or decrease it if the measurement overshoots the setpoint by too much.
-   1. Repeat until you are satisfied with the behaviour.
+   2. Disarm the rover and from the flight log plot the _lateral_acceleration_setpoint_ from [RoverAckermannSetpoint](../msg_docs/RoverAckermannSetpoint.md) and the _measured_lateral_acceleration_ from [RoverAckermannStatus](../msg_docs/RoverAckermannStatus.md) over each other.
```

Comment:
Just FYI, markdown doesn't care and will auto-number. If you're moving points around it is better to not number, if you're referring to specific steps, then it is better to number.

Reply:
Noted, thanks for the info!